src/main/java/com/example/kadai7/form/ComicForm.java

```diff
@@ -0,0 +1,30 @@
+package com.example.kadai7.form;
+
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Positive;
+
+public class ComicForm {
+
+    @NotBlank
+    private String name;
+
+    @Positive
+    private int latest_volume;
```

Comment:
latest_volume(最新刊)とわかりやすくはなったのですが、Javaの命名規則に従うという点がまだ対応しきれていないです。
```suggestion
    private int latestVolume;
```
です。
あわせてGetterとSetterの修正もお願いします！

Reply:
latestVolumeに修正いたしました！

命名規則について、単語区切りは大文字で固定値の場合はアンダーバーで区切るですかね。

Reply:
> 命名規則について、基本的に単語区切りは大文字ですが、固定値の場合はアンダーバーで区切るですかね。

はい。あっています。
定数（=固定値）は正確には大文字のスネークケースですね。
パッケージ名も間違えやすいのでチェックしておくとよいです。
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html